src/Waypoint/WaypointReaderSeeYou.cpp

```diff
@@ -169,20 +169,9 @@ WaypointReaderSeeYou::ParseLine(const TCHAR* line, Waypoints &waypoints)
     iElevation = 5,
     iStyle = 6,
     iRWDir = 7,
-    iRWLen = 8,
-    iFrequency = 9,
-    iDescription = 10,
+    iRWLen = 8
```

Comment:
Why remove the comma? This makes it hard to add more items, for no reason. There was a trailing comma before.